### PR TITLE
doc: releases: v14.2.8 release notes

### PR DESCRIPTION
**Pull request by theanalyst**
Signed-off-by: Abhishek Lekshmanan <abhishek@suse.com>

### CODE REVIEW

File: doc/releases/nautilus.rst [hunk 1]
```diff
@@ -1,3 +1,278 @@
...
+
+* Ceph will now issue a health warning if a RADOS pool as a ``pg_num``
+  value that is not a power of two.  This can be fixed by adjusting
+  the pool to a nearby power of two::
+
+    ceph osd pool set <pool-name> pg_num <new-pg-num>
+
+  Alternatively, the warning can be silenced with::
+
+    ceph config set global mon_warn_on_pool_pg_num_not_power_of_two false
```
Comment (smithfarm) [inline doc/releases/nautilus.rst]: I can see the "pg_num value that is not a power of two" release note in https://ceph.io/releases/v14-2-5-nautilus-released/ 

so it should probably be dropped?

Comment (theanalyst) [reply doc/releases/nautilus.rst]: ack, will do

Comment (theanalyst) [reply doc/releases/nautilus.rst]: dropped

File: doc/releases/nautilus.rst [hunk 2]
```diff
@@ -1,3 +1,268 @@
...
+================
+
+This is the eighth update to the Ceph Nautilus release series. This release
+fixes issues across a range of subsystems. We recommend that all users upgrade
+to this release.
+
+Notable Changes
+---------------
+
+* The following OSD memory config options related to bluestore cache autotuning can now
```
Comment (neha-ojha) [inline doc/releases/nautilus.rst]: Can we include the following?

```suggestion
* The default value of ``bluestore_min_alloc_size_ssd`` has been changed to 4K to improve performance across all workloads.

* The following OSD memory config options related to bluestore cache autotuning can now
```

File: doc/releases/nautilus.rst [hunk 3]
```diff
@@ -1,3 +1,268 @@
...
+  the racks.  In those cases, the configuration value can be increased.
+
+* RGW: a mismatch between the bucket notification documentation and the actual
+  message format was fixed. This means that any endpoints receiving bucket
+  notification, will now receive the same notifications inside an JSON array
+  named 'Records'. Note that this does not affect pulling bucket notification
+  from a subscription in a 'pubsub' zone, as these are already wrapped inside
+  that array.
+
+
```
Comment (neha-ojha) [inline doc/releases/nautilus.rst]: I think https://github.com/ceph/ceph/blob/nautilus/PendingReleaseNotes#L37-L45 got missed?

Comment (smithfarm) [reply doc/releases/nautilus.rst]: @neha-ojha It was dropped because it's a dupe - see https://github.com/ceph/ceph/pull/33670#discussion_r386530362

Comment (neha-ojha) [reply doc/releases/nautilus.rst]: @smithfarm ok, the PR is still in this release https://github.com/ceph/ceph/pull/33670/files#diff-662a7de9ea1019d168ac10d9a198c4c9R136

File: doc/releases/nautilus.rst [hunk 4]
```diff
@@ -1,3 +1,270 @@
...
+  be configured during runtime:
+
+    - osd_memory_base (default: 768 MB)
+    - osd_memory_cache_min (default: 128 MB)
+    - osd_memory_expected_fragmentation (default: 0.15)
+    - osd_memory_target (default: 4 GB)
+
+  The above options can be set with::
+
+    ceph config set global <option> <value>
```
Comment (batrick) [inline doc/releases/nautilus.rst]: Why use `global` here and not `osd`?

File: doc/releases/nautilus.rst [hunk 5]
```diff
@@ -1,3 +1,270 @@
...
+  crush requires 1 replica on each of 3 racks, but there are fewer OSDs in 1 of
+  the racks.  In those cases, the configuration value can be increased.
+
+* RGW: a mismatch between the bucket notification documentation and the actual
+  message format was fixed. This means that any endpoints receiving bucket
+  notification, will now receive the same notifications inside an JSON array
+  named 'Records'. Note that this does not affect pulling bucket notification
+  from a subscription in a 'pubsub' zone, as these are already wrapped inside
+  that array.
+
```
Comment (batrick) [inline doc/releases/nautilus.rst]:
````
CephFS: multiple active MDS forward scrub is now rejected. Scrub currently only is permitted on a file system with a single rank. Reduce the ranks to one via ``ceph fs set <fs_name> max_mds 1``.
````

Comment (batrick) [reply doc/releases/nautilus.rst]: Another:

````
Ceph now refuses to create a file system with a default EC data pool. For further explanation, see: https://docs.ceph.com/docs/master/cephfs/createfs/#creating-pools
````

File: doc/releases/nautilus.rst [hunk 6]
```diff
@@ -1,3 +1,270 @@
...
+  from the target PGs per OSD.  This can be set with a command like
+  ``ceph config set mgr mgr/balancer/upmap_max_deviation 2``.  The default
+  ``upmap_max_deviation`` is 1.  There are situations where crush rules
+  would not allow a pool to ever have completely balanced PGs.  For example, if
+  crush requires 1 replica on each of 3 racks, but there are fewer OSDs in 1 of
+  the racks.  In those cases, the configuration value can be increased.
+
+* RGW: a mismatch between the bucket notification documentation and the actual
+  message format was fixed. This means that any endpoints receiving bucket
+  notification, will now receive the same notifications inside an JSON array
```
Comment (mykaul) [inline doc/releases/nautilus.rst]: "an JSON" -> "a JSON"

File: doc/releases/nautilus.rst (hunk 4 above)
Comment (theanalyst) [reply doc/releases/nautilus.rst]: ack, (was just a C-c C-v from the PendingReleaseNotes)

File: doc/releases/nautilus.rst (hunk 5 above)
Comment (theanalyst) [reply doc/releases/nautilus.rst]: just changed the url to https://docs.ceph.com/docs/nautilus/cephfs/createfs/#creating-pools

**Comment by theanalyst** [comment]
jenkins render docs

**Comment by theanalyst** [comment]
jenkins render docs

File: doc/releases/nautilus.rst (hunk 3 above)
Comment (theanalyst) [reply doc/releases/nautilus.rst]: added this back

**Comment by theanalyst** [comment]
jenkins render docs

File: doc/releases/nautilus.rst [hunk 7]
```diff
@@ -1,3 +1,287 @@
...
+* The MGR now accepts ``profile rbd`` and ``profile rbd-read-only`` user caps.
+  These caps can be used to provide users access to MGR-based RBD functionality
+  such as ``rbd perf image iostat`` an ``rbd perf image iotop``.
+
+* The configuration value ``osd_calc_pg_upmaps_max_stddev`` used for upmap
+  balancing has been removed. Instead use the mgr balancer config
+  ``upmap_max_deviation`` which now is an integer number of PGs of deviation
+  from the target PGs per OSD.  This can be set with a command like
+  ``ceph config set mgr mgr/balancer/upmap_max_deviation 2``.  The default
+  ``upmap_max_deviation`` is 1.  There are situations where crush rules
```
Comment (neha-ojha) [inline doc/releases/nautilus.rst]: sorry, I missed this earlier. 
```suggestion
  ``upmap_max_deviation`` is 5.  There are situations where crush rules
```
https://github.com/ceph/ceph/pull/31956/commits/cc96a4cb0cc65ee0d761656af3689c5740c2c4a9 cc: @dzafman

Comment (theanalyst) [reply doc/releases/nautilus.rst]: tnx, I'll update the blog post as well

Comment (theanalyst) [reply doc/releases/nautilus.rst]: updated blog as well

Comment (neha-ojha) [reply doc/releases/nautilus.rst]: Thanks @theanalyst !

**Comment by theanalyst** [comment]
using the gh autocommit feature had the signed-off-by failure, so squashed the commits, I'll merge once the status is green

**Pull request merged**